agent/bench-scripts/unittests

```diff
@@ -160,6 +160,7 @@ function _dump_logs {
     if [[ ! -z "${rundir}" ]]; then
         _dump_file ${rundir} metadata.log
         _dump_file ${rundir} user-benchmark-summary.json
+        _dump_file ${rundir} SPECjbb.props
```

Comment:
My first thought was "gee, why are we dumping the `props` file in `_dump_logs`"? My second was ... um, we're dumping *one* log file, a json file, an html file, and a set of cmd files, now plus props ... it's really that the function is misnamed. 😆

Reply:
Agreed.  Great task for another PR to solve!

Reply:
Then again, we want to "dump" the legacy tests entirely, and "this, too, shall pass" so maybe it's not worth worrying about too much. It was, however, an amusing observation I couldn't help sharing ... but will now quietly mark resolved. 😁